HTTP UA Client Hint headers (#11191)
mdn/content: files/en-us/web/http/headers/sec-ch-ua-bitness/index.md

The **`Sec-CH-UA-Bitness`** [user agent client hint](/en-US/docs/Glossary/Client_hints#user-agent_client_hints) request header provides the "bitness" of the user-agent's underlying CPU architecture. 
This is the size in bits of an integer or memory address—typically 64 or 32 bits.

This might be used by a server, for example, to select and offer the correct binary format of an executable for a user to download.

<table class="properties">
  <tbody>
    <tr>
      <th scope="row">Header type</th>
      <td>
        {{Glossary("Request header")}},
        {{Glossary("Client hints","Client hint")}}
      </td>
    </tr>
    <tr>
      <th scope="row">{{Glossary("Forbidden header name")}}</th>
      <td>yes</td>
    </tr>
  </tbody>
</table>

## Syntax

```http
Sec-CH-UA-Bitness: <bitness>
```

## Directives

- `<bitness>`
  - : A string indicating the underlying platform architecture bitness, such as: `"64"`, `"32"`.

## Examples

A server requests the `Sec-CH-UA-Bitness` header by including the {{HTTPHeader("Accept-CH")}} in a _response_ to any request from the client, using the name of the desired header as a token:

```http
HTTP/1.1 200 OK
Accept-CH: Sec-CH-UA-Bitness
```

> The client may then opt-in to providing the hint, and add the `Sec-CH-UA-Bitness` header to subsequent requests.

Author: ```suggestion
The client may choose to provide the hint, and add the `Sec-CH-UA-Bitness` header to subsequent requests.
```
